.travis.yml

```diff
@@ -46,15 +52,26 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gtk+3 cairo atk; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/opt/X11/lib/pkgconfig; fi
+  - if [[ "$ARM" == "1" ]]; then rustup target add armv7-unknown-linux-gnueabihf; fi
 script:
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ] && ! [ "$ARM" == "1" ]; then
     make regen_check;
     fi
-  - cargo doc --features "dox"
-  - cargo test --features "subclassing"
+  - if ! [ "$ARM" == "1" ]; then
+    cargo doc --features "dox";
+    fi
+  - if ! [ "$ARM" == "1" ]; then
+    cargo test --features "subclassing";
+    fi
+  - if [ "$ARM" == "1" ]; then
+    PKG_CONFIG_ALLOW_CROSS=1 cargo build $OTHER_TARGET;
```

Comment:
Simply because it's easier to spot errors when reading CI output. Actually, there is a way to have "stages" in travis but I don't have enough time to write a clean CI script for the moment.